ansible/roles/runner/tasks/setup.yml

```diff
@@ -22,23 +22,52 @@
   with_items:
     - tuned
     - vagrant
+    - vagrant-sshfs
     - vagrant-libvirt
     - libguestfs-tools-c  # required for vagrant package command
     - ansible  # TODO requires >= 2.3
     - nfs-utils
     - git
     - libvirt
+    - libvirt-devel
+    - ruby-devel
     - rsync
     - createrepo
     - gzip
     - gcc
     - python3-devel
     - python3-pip
+    - python3-winrm
+    - python2-winrm
+    - NetworkManager
+    - python-dnf
     - redhat-rpm-config
     - crontabs
   notify:
     - restart_nfs
 
+# FIXME: something is breaking python3-six installation,
```

Comment:
@netoarmando, I have tested this and it doesn't work:

task was executed: 
TASK [runner : force reinstall python3-six] ******************************************************************************************
changed: [10.8.247.3]

but:
Feb 05 11:02:48 ad bash[28623]:     import urllib3
Feb 05 11:02:48 ad bash[28623]:   File "/usr/lib/python3.6/site-packages/urllib3/__init__.py", line 8, in <module>
Feb 05 11:02:48 ad bash[28623]:     from .connectionpool import (
Feb 05 11:02:48 ad bash[28623]:   File "/usr/lib/python3.6/site-packages/urllib3/connectionpool.py", line 11, in <module>
Feb 05 11:02:48 ad bash[28623]:     from .exceptions import (
Feb 05 11:02:48 ad bash[28623]:   File "/usr/lib/python3.6/site-packages/urllib3/exceptions.py", line 2, in <module>
Feb 05 11:02:48 ad bash[28623]:     from .packages.six.moves.http_client import (
Feb 05 11:02:48 ad bash[28623]: ModuleNotFoundError: No module named 'urllib3.packages.six'
Feb 05 11:02:48 ad systemd[1]: prci.service: Main process exited, code=exited, status=1/FAILURE
Feb 05 11:02:48 ad systemd[1]: prci.service: Failed with result 'exit-code'.

Surprisingly, I executed the command by hand and worksforme... so I think we should move this tasks to the end of the playbook.

Reply:
Moving this WA to deploy_pr_ci.yml playbook is the solution. This step is now validated:

ad                : ok=42   changed=13   unreachable=0    failed=0

systemctl status prci
Active: active (running)